vpd_tool.cpp

```diff
@@ -107,12 +107,18 @@ int main(int argc, char** argv)
 
     try
     {
-        if (objectPath.empty())
+        if ((*writeFlag || *readFlag || *dumpObjFlag) && *object &&
```

Comment:
why do you need these "(*writeFlag || *readFlag || *dumpObjFlag)" 
just check if object and if it is empty , as we cannot keep adding to this whenever new commands come in.

Reply:
updated.